Backend/models/TestReportModel.js

```diff
@@ -3,58 +3,56 @@ const mongoose = require("mongoose");
 const testReportSchema = new mongoose.Schema({
   _id: mongoose.Schema.Types.ObjectId,
 
-    user_id: {
```

Comment:
This is cool if we inform web team (click resolve if we checked with mobile team)